src/topology/category/CompHaus/default.lean

```diff
@@ -169,23 +170,27 @@ def limit_cone {J : Type u} [small_category J] (F : J ⥤ CompHaus.{u}) :
 { X :=
   { to_Top := (Top.limit_cone (F ⋙ CompHaus_to_Top)).X,
     is_compact := begin
-      dsimp [Top.limit_cone],
+      show compact_space ↥{u : Π j, (F.obj j) | ∀ {i j : J} (f : i ⟶ j), (F.map f) (u i) = u j},
```

Comment:
All the changes to this proof are merely to speed it up so that it doesn't time out in my VScode.